packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -66,15 +66,21 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
                 display: "block",
                 height: "4px",
                 width: "32px",
-                transition: "all 0.2s ease-in-out",
+                transition: "all 0.05s ease-in-out",
             },
-            "&:hover": {
-                "&::before": {
-                    opacity: "0.5",
+            "&:not($carousel_sequenceIndicator__active)": {
+                "&:hover": {
+                    "&::before": {
+                        opacity: "0.5",
+                    },
                 },
             },
         },
-        carousel_sequenceIndicator__active: {},
+        carousel_sequenceIndicator__active: {
+            "&::before": {
+                opacity: "1",
+            },
+        },
```

Comment:
Can we do this in a different PR? The example data is just a "chore", whereas this is a "fix". Seperating these PR's will keep things clear in the changelog and history.

Reply:
Removed style changes from this PR